tests/integration/new_relations/test_new_relations.py

```diff
@@ -37,7 +37,7 @@
 INVALID_EXTRA_USER_ROLE_BLOCKING_MESSAGE = "invalid role(s) for extra user roles"
 
 
-@pytest.mark.group(1)
+@pytest.mark.group("main_tests")
```

Comment:
Something like high availability might be a more meaningful label.

Reply:
ha_tests?

Reply:
oops, this is different from K8s: we're on `test_new_relations`, not `self_healing` - what about `new_relations_tests`?

Reply:
merging this so it gets out of the way - we can always create follow ups if better ideas come!